api/src/firstrun.py

```diff
@@ -99,16 +109,82 @@ def create_admin(admins):
 
 
 def create_members():
-    banner(RED, "TDOO: Creating Fake Members")
+    banner(RED, "TODO: Creating Fake Members")
```

Comment:
You can update this message too and remove the TODO since the TODO is done by this PR